### PR TITLE
set next button as loading until next page loads

### PR DESCRIPTION
**Pull request by dan13ram**
Closing #403

### CODE REVIEW

File: packages/web/components/Setup/SetupDone.tsx [hunk 1]
```diff
@@ -20,10 +21,14 @@ export const SetupDone: React.FC = () => {
       >
         {user?.player && <PlayerTile player={user.player} />}
         <MetaButton
-          onClick={() => router.push('/')}
+          onClick={() => {
+            setLoading(true);
```
Comment (dysbulic) [inline packages/web/components/Setup/SetupDone.tsx:25]: ¿Have you seen a delay on this component updating state?

I'm pretty sure this update is fast enough that we don't need the `loading` check, but it shouldn't hurt anything to have this.

File: packages/web/components/Setup/SetupUsername.tsx [hunk 2]
```diff
@@ -33,14 +35,16 @@ export const SetupUsername: React.FC<SetupUsernameProps> = ({ username, setUsern
       if (error.message.includes('Uniqueness violation')) {
         errorDetail = 'This username is already taken 😢';
       } else if (error.message.includes('username_is_valid')) {
-        errorDetail = 'A username can only contain letters, numbers, and dashes.';
+        errorDetail =
```
Comment (dysbulic) [inline packages/web/components/Setup/SetupUsername.tsx:38]: I try to always have a character that marks the end of an indentation block.

```suggestion
        errorDetail = (
          'A username can only contain letters, numbers, and dashes.'
        );
```

**Review by dysbulic — APPROVED** [review]
This looks pretty straightforward. You added a `loading` state variable that maintains the `isLoading` property as `true` until the page load finishes.

I have no significant suggestions.

File: packages/web/components/Setup/SetupDone.tsx (hunk 1 above)
Comment (dan13ram) [reply packages/web/components/Setup/SetupDone.tsx:25]: Yeah. The delay is noticeable when testing in `dev` mode, but it won't be in prod.
Like you said though, it won't hurt to keep it.

**Pull request merged**